### ISSUE TITLE
Request and Install Certificate

### ISSUE THREAD

**Issue by fredrik-annytab** [sjkp/letsencrypt-siteextension#172]
Tried to create one certificate for 66 domain names but the request timed out after 230 seconds. There is a 230 second (i.e. a little less than 4 mins) timeout for requests that are not sending any data back.  

500 - The request timed out. The web server failed to respond within the specified time.

The task continued to run server side and the certificate was installed.

A solution to this might be to return a message immediately like: The certificate is being installed in the background and it might take some time, check back later.

**Comment by sjkp** [comment]
Thanks for the heads up, I will leave it open as an future enhancement. I'm always amazed by these to me crazy scenarios that the extension is used for, but please note that Lets Encrypt only supports up to a 100 domains per certificate.

**Comment by modemgeek** [comment]
totally agree.  we are securing about 200+ hostnames with 200+ certificates per webapp.  We run into these timeouts when a bunch are up for renewal.  However, it does work and will eventually renew them all

**Comment by sjkp** [comment]
Btw, did you make a ticket for MS, because the timeout and long times you see are due to slow Azure APIs right? Maybe they can improve their backend.

**Comment by modemgeek** [comment]
Yes I have and they have made some improvements. One thing I noticed though
is that if you have a URL domain that cannot renew, the renewal function
stops. Is this how it is supposed to work or should it continue trying to
renew the remaining certs?

**Comment by RIKIKU** [comment]
@modemgeek It sounds like you have made it create/renew a separate SSL cert per domain name, is this correct?  If so, mind sharing how you did this? I also run WebApps with multiple domain names per app and it would be nice to make it create a cert per hostname.
